wdio/specs/news-item.js

```diff
@@ -20,4 +20,23 @@ describe('Municipality news item page', () => {
     browser.waitForVisible('div=إدارة مجموعات تعلن: الفرصة الأخيرة لإغلاق ديون ضريبة الأملاك للسكان. عجل تسوية الديون الخاصة بك وتجنب العقوبات. يسري هذا العرض حتى -8.4.17.');
   });
 
+  it('should not show an expired news item after running cron', () => {
```

Comment:
I don't think WDIO should test this. IMO simpleTest is more appropriate. If time doesn't permit, leave as is.

Reply:
I'll change to a SimpleTest.